src/components/ui/Footer.js

```diff
@@ -0,0 +1,67 @@
+'use client'
+
+import React from 'react';
+import Image from 'next/image';
+import FYLPLogo from '/public/assets/images/LOGO_FYLP.png';
+import Link from 'next/link';
+import FacebookRoundedIcon from '@mui/icons-material/FacebookRounded';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import Socials from '/src/components/ui/Socials.js'
+
+
+const Footer = () => {
+  return (
+   
+    <div id='main' className='w-full flex flex-col px-[50px] md:px-[100px] items-center'>
+  
+      <div className='bg-[#202020] opacity-15 w-full h-[2px]'></div>
+          <div id='logo' className='flex flex-col items-center mt-10'> 
+            <Image src={FYLPLogo} alt="FYLP Logo" width={150} height={72} className='block xl:hidden'/>
+            <p className='font-poppins text-xs w-[220px] h-[40px] mt-2  font-thin block xl:hidden text-center xl:text-left'>Fostering Leadership and Innovation for a Better Tomorrow</p>
+          </div>
+      <div id='content' className='flex flex-col lg:flex-row mt-[52px] w-full ml-auto mr-auto items-center lg:items-start lg:justify-between'>
+        <div id='left' className='  flex flex-col lg:flex-row'>
+          <div id='logo' className='hidden xl:block'> 
+            <Image src={FYLPLogo} alt="FYLP Logo" width={150} height={72} />
+            <p className='font-poppins text-xs w-[200px] h-[40px] mt-2 text-left font-thin'>Fostering Leadership and Innovation for a Better Tomorrow</p>
+          </div>
+          <div id='links' className='w-[200px] h-[180px] ml-0 xl:ml-20 flex flex-col space-y-3 text-center lg:text-left mb-[25px] lg:mb-0'>
+            <p className='font-poppins font-bold text-[15px] w-[200px] h-[30px]'>Quick Links</p>
+            <Link href="#" className='font-poppins text-sm hover:text-[#FD1E0A] transition-colors duration-300'>Home</Link>
+            <Link href="#" className='font-poppins text-sm hover:text-[#FD1E0A] transition-colors duration-300'>About Us</Link>
+            <Link href="#" className='font-poppins text-sm hover:text-[#FD1E0A] transition-colors duration-300'>FAQ</Link>
```

Comment:
Move the font-poppins property to the div container wrapping the texts, para ma apply ang font-poppins sa tanan, and you dont have to manually put font-poppins everytime